spec/requests/request_from_logux_server_spec.rb

```diff
@@ -87,4 +89,109 @@
       expect { request_logux }.to change { logux_store.size }.by(1)
     end
   end
+
+  # rubocop: disable RSpec/NestedGroups
+  # TODO: refactoring; may be move this cases to separated file
```

Comment:
@moofkit , what do you think about doing it with the next pull request? 😉

Reply:
@dsalahutdinov, yeah of course 👍